src/modules/analytics/track.ts

```diff
@@ -300,8 +299,10 @@ addPayload(CREATE_COLLECTION_FORUM_POST_FAILURE, 'Create forum post error')
 
 // ENS analytics
 add(SET_ENS_RESOLVER_SUCCESS, 'Set ENS Resolver', action => {
-  const { payload } = action as FetchTransactionSuccessAction
-  const { ens, resolver, address } = payload.transaction.payload
+  // TODO: Use correct Type
+  const { payload } = action as any
```

Comment:
There seems to have been changes in the object schema and transaction does not exist inside payload anymore. This caused a blocking error on the flow. This is a simple patch but should be investigated more

Reply:
Used an util function from dapps to solve this